datafusion-postgres-cli/src/main.rs

```diff
@@ -123,7 +130,7 @@ async fn setup_session_context(
             .register_csv(table_name, table_path, CsvReadOptions::default())
             .await
             .map_err(|e| format!("Failed to register CSV table '{table_name}': {e}"))?;
-        println!("Loaded {table_path} as table {table_name}");
+        info!("Loaded {table_path} as table {table_name}");
```

Comment:
It's ok to have print in cli module. Otherwise user will need additional configuration to see the output